src/balancer/OceanPool.ts

```diff
@@ -630,7 +631,7 @@ export class OceanPool extends Pool {
       this.logger.error('ERROR: OCEAN approve failed')
       return null
     }
-    return this.swapExactAmountIn(
+    const tx = this.swapExactAmountIn(
```

Comment:
interesting that `this.swapExactAmountIn` does not need `await` to work in my local test

Reply:
it should work, but never the less..

Reply:
but typings say they are promises, so they should get an `await`

Reply:
fun fact is that swapExactAmountIn returns a promise, so as long as you do await BuyDT, you are resolving the promise

Reply:
same here, i can sell a dt, whilst on prod i can't